homeassistant/components/mqtt/server.py

```diff
@@ -85,6 +98,6 @@ def generate_config(hass, passwd):
         username = None
         password = None
 
-    client_config = ('localhost', 1883, username, password, None, PROTOCOL_311)
+    client_config = ('localhost', 1883, username, password, None, '3.1.1')
```

Comment:
In the end, `server.py` will contain all HBMQTT specific code, with generate_config and the schema together (both very dependant on the hbmqtt library)

Will move the import inside the functin, or better yet, import from hbmqtt if possible 👍